src/test/rustdoc-ui/intra-doc/feature-gate-pointer.rs

```diff
@@ -0,0 +1,3 @@
+// gate-test-intra_doc_pointers
```

Comment:
I think this might need to be annotated with `// check-fail`, because CI is failing. Also, maybe this test should be named `feature-gate-intra-doc-pointers` to be consistent with the feature name?

Reply:
CI is failing because I forgot to add a feature gate to another test. I renamed the file to match the feature gate.